test/test_json_util.py

```diff
@@ -71,6 +71,11 @@ def round_trip(self, doc, **kwargs):
     def test_basic(self):
         self.round_trip({"hello": "world"})
 
+    def test_loads_bytes(self):
+        string = rb'{"hello": "world"}'
+        for type in [bytes, bytearray]:
+            json_util.loads(type(string))
```

Comment:
In this case using a list makes the code less understandable from a type perspective. I suggest:
```
            json_util.loads(b'{"hello": "world"}')
            json_util.loads(bytearrary(b'{"hello": "world"}'))
```

Reply:
(with assertions that the loaded value is as expected).

Reply:
Done.